Library/Homebrew/test/spec_helper.rb

```diff
@@ -180,6 +180,10 @@
     skip "Unzip is not installed." unless which("unzip")
   end
 
+  config.before(:each, :dev_on_linux) do
```

Comment:
The advantage of this helper is that should we decide to remove support for casks on Linux completely, even for developers, we'd be able to just remove this and replace all instances of `:dev_on_linux` with `:needs_macos`.

Reply:
I was initially unsure about this but I really like this reasoning!